src/client/src/apps/MainRoute/routes/styled.tsx

```diff
@@ -4,7 +4,7 @@ import { rules } from 'rt-styleguide'
 export const OverflowScroll = styled.div`
   overflow-y: scroll;
   ${rules.touchScroll};
-  height: 100%;
+  height: 100vh;
```

Comment:
I think this was added because of how the layout works when running in Platform API mode, but this messes up the scroll height of the lives rates in the web app.